README.md

```diff
@@ -151,3 +151,39 @@ To test the NGINX Kubernetes Gateway run:
 ```
 make unit-test
 ```
+
+# Release Process for NGINX Kubernetes Gateway
+
+### NGINX Kubernetes Gateway will use semantic versioning for releases described in semver.org.. 
```

Comment:
### NGINX Kubernetes Gateway uses semantic versioning for releases (https://semver.org/).